Scenes/Player/player.gd

```diff
@@ -1,38 +1,91 @@
 extends CharacterBody3D
 
 
-const SPEED = 5.0
-const JUMP_VELOCITY = 4.5
+const SPEED_DEFAULT : float = 5.0
+const CAMERA_ROTATION_SPEED_DEFAULT : float = 3.0
+const JUMP_VELOCITY_DEFAULT : float = 9.0
```

Comment:
Changed const values to defaults. Possibly better to simply make these export variables with default values based on the scene?